<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>280</width>
      <height>166</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="reserveButton">
         <property name="text">
          <string>reserve</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resizeButton">
         <property name="text">
          <string>resize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="assignButton">
         <property name="text">
          <string>assign</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="push_backButton">
         <property name="text">
          <string>push_back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="empalce_backButton">
         <property name="text">
          <string>emplace_back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="emplaceButton">
         <property name="text">
          <string>emplace</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="sizeButton">
         <property name="text">
          <string>size</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pop_backButton">
         <property name="text">
          <string>pop_back</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="atButton">
         <property name="text">
          <string>at</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="eraseButton">
         <property name="text">
          <string>erase</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="capacityButton">
         <property name="text">
          <string>capascity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="insertButton">
         <property name="text">
          <string>insert</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="max_sizeButton">
         <property name="text">
          <string>max_size</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>193</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>result:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="resultEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>197</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>vector:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="vectorEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>187</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>index:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="indexEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>190</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>value:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="valueEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>additional size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="forresizeEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
